AUTHORS.rst

```diff
@@ -0,0 +1,30 @@
+*******************
+Authors and Credits
+*******************
+
+The primary contributors to PyERFA are
+
+* Julien Woillez, who created the first Jinja2_ template, for wrappers
+  based on an array iterator.
+* Erik Tollerud, who helped with the initial wrapping and with integration.
+* Michael Droettboom, who helped with the initial wrapping.
+* Marten van Kerkwijk, who changed the wrappers to use Numpy_
+  universal functions.
+* Antonio Valentino, who created PyERFA from the implementation in
+  Astropy_.
+* Thomas Robitaille, who helped make most steps possible.
```

Comment:
Perhaps an overstatement of my contributions ;) Maybe you can just say something like 'who helped with the transition to a standalone package' or something like that

Reply:
You may have forgotten, but you helped with earlier steps too! I'll rephrase "helped with many of the steps".